File: Triatlon-Java/TriatlonPersistance/src/main/java/repository/interfaces/InscriereRepository.java
package repository.interfaces;

import domain.*;
import repository.Repository;

import java.util.List;

public interface InscriereRepository extends Repository<Long, Inscriere> {
    List<Long> findParticipantsByProba(Long idProba);
    List<Inscriere> findInscriereByProbaAndParticipant(Long idProba, Long idParticipant);
}

File: Triatlon-C#/Triatlon/repository/interfaces/InscriereRepository.cs
using System.Collections.Generic;
using ConcursTriatlon.domain;
using ConcursTriatlon.repository;

namespace ConcursTriatlon.repository.interfaces
{
    public interface InscriereRepository : Repository<long, Inscriere>
    {
        List<long> findParticipantByProba(long idProba);
        List<long> findInscriereByProbaAndParticipant(long idProba, long idParticipant);

    }
}

